src/components/List/List.js

```diff
@@ -18,29 +19,46 @@ export const timeDifference = (timestampInMilli) => {
   const difference = date - timestampInMilli;
   const comparison = difference < dayInMilli;
 
```

Comment:
Did you elect to comment out to keep the console clear, but retain these in comments for future reference? I'd recommend deleting at this point but if they're useful, tell me so and feel free to keep them around.